lib/structuraid_core/designers/footings/steps/centric_isolated/compute_required_rebar_ratio.rb

```diff
@@ -0,0 +1,85 @@
+require 'interactor'
+
+module StructuraidCore
+  module Designers
+    module Footings
+      module Steps
+        module CentricIsolated
+          # Runs the structural bending analysis and design for a centric isolated footing, generating the required reinforcement ratio and checking that the provided one is enough
+          class ComputeRequiredRebarRatio
+            include Interactor
+
+            # @param footing [StructuraidCore::Elements::Footing] The footing to be designed
+            # @param load_scenario [StructuraidCore::Loads::Scenarios::Footings::CentricIsolated] The load scenario to be considered
+            # @param analysis_direction [Symbol] The direction for which the analysis has to be run. Should be either :length_1 or :length2
+            # @param design_code [StructuraidCore::DesignCodes] The design code to be used
+            # @param steel [StructuraidCore::Materials::Steel] The rebar's material
+            def call
+              add_analysis_results_to_context
+              context.analysis_results[:bending_momentum] = compute_flexural_moment
+              context.analysis_results[:required_reinforcement_ratio] = compute_required_flexural_reinforcement_ratio
+            rescue DesignCodes::RequirementNotFulfilledError => e
+              context.fail!(message: e.message)
+            end
+
+            private
+
+            def add_analysis_results_to_context
+              context.analysis_results = {
+                bending_momentum: nil,
+                required_reinforcement_ratio: nil
+              }
+            end
+
+            def compute_flexural_moment
+              analysis = Engineering::Analysis::Footing::CentricIsolated.new(
+                footing:,
+                load_from_column: load_scenario.total_ultimate_load,
+                section_direction: analysis_direction
+              )
+
+              - analysis.bending_solicitation
```

Comment:
I think this `-` operation should be inside the analysis class, not here

Reply:
You are right, let's use the result of #bending_solicitation; we will need it for the other footings cases, and just let's use de absolute the result for the rebar ratio computation